roles/import_image_and_run_gcp_instance/README.md

```diff
@@ -0,0 +1,104 @@
+* **import_image_and_run_gcp_instance_labels** (dict): (Optional) Labels to apply to this instance. A list of key->value pairs.
+* **import_image_and_run_gcp_instance_metadata** (dict): (Optional) The metadata key/value pairs to assign to instances that are created from this template. These pairs can consist of custom metadata or predefined keys.
+* **import_image_and_run_gcp_instance_machine_type** (str): (Optional) A reference to a machine type which defines VM kind.
+* **import_image_and_run_gcp_instance_scheduling** (dict): (Optional) Sets the scheduling options for this instance.
+* **import_image_and_run_gcp_instance_service_accounts** (list): (Optional) A list of service accounts, with their specified scopes, authorized for this instance. Only one service account per VM instance is supported.
+* **import_image_and_run_gcp_instance_tags** (list): (Optional) A list of tags to apply to this instance. Tags are used to identify valid sources or targets for network firewalls and are specified by the client during instance creation. The tags can be later modified by the setTags method. Each tag within the list must comply with RFC1035.
+* **import_image_and_run_gcp_instance_network_interfaces** (list): (Required) An array of configurations for this interface. This specifies how this interface is configured to interact with other network services, such as connecting to the internet. At least one network interface is required. For a full list of parameters visit https://docs.ansible.com/ansible/latest/collections/google/cloud/gcp_compute_instance_module.html#parameter-network_interfaces. It must exist in the region the instance is created.
+* **import_image_and_run_gcp_zone** (str): (Optional) A reference to the zone where the compute machine resides. If not set, it defaults to 'us-central1-a'.
+
+Dependencies
+------------
+
+- role: [gcp_setup_credentials](../gcp_setup_credentials/README.md)
+
+Example Playbook
+----------------
+This role can be used together with the [cloud.gcp_ops.clone_on_prem_vm](../clone_on_prem_vm/README.md) role as shown below.
+
+    - hosts: localhost
+      gather_facts: false
+
+      vars:
+        on_prem_source_vm_name: "ubuntu-guest"
+        on_prem_vm_image_name: "ubuntu-guest-image"
+        bucket_name: "vm-s3-bucket"
+        instance_name: "vm-clone"
+        local_image_path: "~/images/"
+        kvm_host:
+          name: kvm
+          ansible_host: 192.168.1.117
+          ansible_user: vagrant
+          groups: "libvirt"
+          ansible_ssh_private_key_file: ~/.ssh/id_rsa.pub
+        machine_type: "t2.micro"
+        import_image_name: "import-clone"
+        network_interfaces:
+          - network: "network-instance-clone-vm"
+            access_configs:
+            - name: External NAT
+              nat_ip:
+                address: "35.188.61.177"
+              type: ONE_TO_ONE_NAT
+
+      tasks:
+        - name: Add host to inventory
```

Comment:
same here, move this to an inventory file (for controller use case it is more friendly)